pkg/project/options.go

```diff
@@ -59,7 +60,13 @@ func FromConfig(c *Config) (*Project, error) {
 	}
 
 	if len(p.Functions) == 0 {
-		return nil, fmt.Errorf("no functions were found within match on handlers '%+v' in dir '%s', try a new pattern", c.ConcreteHandlers, p.Dir)
+		handlerMatches := lo.Reduce(c.ConcreteHandlers, func (agg string, handler *HandlerConfig, idx int) string {
```

Comment:
This was an output bug: it would output the address instead of the actual handler name.